File: src/main/java/com/example/drycleaner/dto/dry_cleaner/response/DryCleanerResponseDto.java
package com.example.drycleaner.dto.dry_cleaner.response;

import com.example.drycleaner.dto.service.response.ServiceResponseDto;
import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.experimental.FieldDefaults;

import java.util.List;
import java.util.UUID;

@Data
@FieldDefaults(level = AccessLevel.PRIVATE)
@AllArgsConstructor
public class DryCleanerResponseDto {
    UUID dryCleanerId;
    String name;
    String location;
    String description;
    List<ServiceResponseDto> services;
    List<String> imageUrl;
    UUID userId;
}

File: src/main/java/com/example/drycleaner/dto/dry_cleaner/response/DryCleanerResponseDto.kt
package com.example.drycleaner.dto.dry_cleaner.response

import com.example.drycleaner.dto.service.response.ServiceResponseDto

data class DryCleanerResponseDto(
    val id: String?,
    val name: String?,
    val location: String?,
    val description: String?,
    val services: List<ServiceResponseDto>?,
    val imageUrls: List<String>?,
    val userId: String?
)